src/Inventory/Conf.php

```diff
@@ -216,8 +217,15 @@ protected function importContentFile(Request $inventory_request, $path, $content
             $inventory_request->handleContentType($mime);
             $inventory_request->handleRequest($contents);
             if ($inventory_request->inError()) {
+                $response = $inventory_request->getResponse();
+                if ($inventory_request->getMode() === Request::JSON_MODE) {
+                    $response = json_decode($inventory_request->getResponse());
```

Comment:
This will return an object, and will probably produce an error.
Unless I am wrong, only `message` entry of response should be displayed here.